content/blog/pycon-india-tickets-2018.md

```diff
@@ -0,0 +1,34 @@
+Title: PyCon India Tickets 2018
+Date: 2018-05-31 12:45
+Category: 2018
+Slug: pycon-india-tickets-2018
+Summary: Details about the different tickets for PyCon India 2018.
+
+PyCon India 2018, the tenth edition of the premier conference in India on using and developing the Python programming language is conducted annually by the Python developer community has started ticket sales for the mega event that is scheduled from 5th - 9th October 2018. 
+
+The conference will be having a tight schedule of workshops,  keynotes, talks, workshops, and dev-sprints strung together through 3 different tracks. Any person who has a conference ticket is eligible to attend the Devsprints. The ticket structure is as follows:- 
```

Comment:
For the devsprints can you mention that though the tickets will be free we will have registrations to manage the logistics ? See FAQs for more details.

Reply:
That would be better. Implementing.